### DISCUSSION TITLE
Guidance on using Remix with Begin

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [beginner-corp/begin-community#111] [Frameworks]
Remix can target both architect and deno (experimental). So I'm wondering how we might use Begin and Remix together?

**Comment by tbeseda** [comment]
Hi @fergusmeiklejohn -- good to talk to you on Discord, too.

I have these parts (Remix and Deno) working separately, so I'll work on a sample repo that can be deployed to Begin. I'll update here with progress.
